database_collections.go

```diff
@@ -86,6 +86,8 @@ type CreateCollectionOptions struct {
 	IndexBuckets int `json:"indexBuckets,omitempty"`
 	// Specifies how keys in the collection are created.
 	KeyOptions *CollectionKeyOptions `json:"keyOptions,omitempty"`
+	// This field is used for internal purposes only. DO NOT USE.
+	DistributeShardsLike string `json:"distributeShardsLike,omitempty"`
```

Comment:
This doesn't seem to belong into this PR.

Reply:
See note in summary: "Note. This PR extends #62"

Reply:
OK LGTM